coralproject/talk
# Rename "single comment view" to "zen mode"

## What does this PR do?

Introduces the concept of "zen mode", which is more fun/descriptive than just "single comment view". 

## How do I test this PR?

- Go to a mod queue
- Hit `z`, it will toggle single comment view
- Test accepting/rejecting comments with shortcuts
- Shortcut menu should say `z` for `Zen mode`
- Docs and translations are updated too
